content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -79,7 +79,7 @@ The webhook server in the e2e test is deployed in the Kubernetes cluster, via
 the [deployment API](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#deployment-v1-apps).
 The test also creates a [service](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#service-v1-core)
 as the front-end of the webhook server. See
-[code](https://github.com/kubernetes/kubernetes/blob/v1.15.0/test/e2e/apimachinery/webhook.go#L301).
+[code](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/apimachinery/webhook.go#L748).
```

Comment:
Hi. Can we fix that to still be a specific branch, but update it to a more recent release (eg: v1.22.0)?

Linking to _master_ isn't stable; linking to a specific branch is, but requires effort to keep it updated as we make new releases.

Reply:
Thanks, @sftim  for looking at this.

> Linking to master isn't stable; linking to a specific branch

Agree with you. we can link it to a specific version (as code modified).

>  but requires effort to keep it updated as we make new releases.

or can we mention  **{{< param "fullversion" >}}** instead of hardcoded the specific version and its value can be derived from here: https://github.com/kubernetes/website/blob/main/config.toml#L143  for every k8s release we have corresponding docs release, can we use that version here.

What do you suggest?

Reply:
> What do you suggest?

Fix that to still reference a specific branch, but update it to a more recent release; ie v1.22.0

We can't use `{{< param "fullversion" >}}` because we then don't know that the **line number** (fragment identifier) in the hyperlink remains accurate.

Reply:
Thanks, @sftim for the suggestion.
I will change it to the most recent version i.e. v1.22.0

Reply:
Hi @sftim 
Addressed the comment, please take a look!

Reply:
Hi @jpbetz @liggitt @jimangel @sftim 
PTAL